src/main.test.js

```diff
@@ -92,30 +92,30 @@ test("Untyped environmental variable", () => {
 
 test("Multiple invalid variables and types", () => {
   const testEnv3 = "DATABASE_URLαα: αα\nADMIN_EMAIL:\n1DEBUG";
-  expect(envSpecToHTML(testEnv3)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv3)).resolves.toEqual("Error:Wrong Syntax");
 });
 
 test("Wrong Syntax for restricted choices", () => {
   const testEnv = "DATABASE_URL\nADMIN_EMAIL:email\nDATA:[0,";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
 });
 
 test("Wrong Syntax for restricted choices2", () => {
   const testEnv = "DEBUG: [0, ]";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
```

Comment:
We better test `envSpec.parse` here instead and we better use [`.rejects`](https://jestjs.io/docs/en/tutorial-async#rejects) since syntax errors make promises get rejected.